### PR TITLE
Feat/add footer to main layout

### PR DESCRIPTION
**Pull request by robertoHeCi**
### Describe your changes:
 - Create responsive footer
 - Add i18n translations
 - Improve theme adding more space variables
 - Add Footer test

### Issue ticket number and link:
 
- Link:  #42 

### Checklist before requesting a review:

https://github.com/onebeyond/shieldjs/assets/58053533/c0ac25c4-ad9d-4cb5-85af-716bd4d4464f
 

- [x] Your code builds clean without any errors or warnings
- [x] I have performed a self-review of my code
- [X] If it is a core feature, I have added thorough tests.

### CODE REVIEW

File: src/components/Footer/Footer.styles.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  '@md': {
+    gap: '3.8rem'
+  }
+})
+
+export const MenuListItem = styled('li', {
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  marginBottom: '$4'
```
Comment (Donivanes) [inline src/components/Footer/Footer.styles.tsx:52]: Also I did `mt mb mr ml`

File: src/layout/MainLayout/MainLayout.styles.tsx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { styled } from '@/stitches.config'
+
+export const MainWrapper = styled('div', {
+  display: 'grid',
+  gridTemplateRows: 'auto 1fr auto',
+  minHeight: '100vh'
+})
```
Comment (Donivanes) [inline src/layout/MainLayout/MainLayout.styles.tsx:7]: I would like to have a chat to discuss this way of having the MainLayout. This way you will always need 3 elements on your layout, but imagine if you don't need the navbar or the footer. I think that with flex is more versatile

**Review by Donivanes — COMMENTED** [review]
Nice work! 🚀  I've left a few comments.

File: src/layout/MainLayout/MainLayout.styles.tsx (hunk 2 above)
Comment (Donivanes) [reply src/layout/MainLayout/MainLayout.styles.tsx:7]: We can have a call if you want

**Review by acalvom — APPROVED** [review]

**Pull request merged**